datadog-checks-tests-helper/tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+minversion = 2.0
+envlist =
+    py27
+    flake8
+
+[testenv:py27]
+deps =
+  pytest
+  -rtests/requirements.txt
+commands =
+  pip install --require-hashes -r requirements.txt
+  pytest
```

Comment:
There aren't any tests here are there? I don't think this bit will do anything. (Strongly in favor of keeping the flake8 but this just seems to be extraneous)

Reply:
?  "pytest" is what executes the tests, yes?

Reply:
Yes but there are no tests in the `datadog-checks-tests-helper` package